utility/src/main/java/org/oppia/android/util/parser/html/BUILD.bazel

```diff
@@ -46,6 +46,7 @@ kt_android_library(
     name = "list_item_leading_margin_span",
     srcs = [
         "ListItemLeadingMarginSpan.kt",
+        "LeftAlignedSymbolsSpan.kt",
```

Comment:
@TanishMoral11, to answer your question regarding adding this class to the build file:
1. Sometimes a kt lib can have multiple source files like you have added here, and for `tag_handlers`, but it depends on usecase. E.g., if you want to use `LeftAlignedSymbolsSpan` in a different part of the codepase, you will need to add a dependency on the lib, `list_item_leading_margin_span`, which will provide both `ListItemLeadingMarginSpan` and `LeftAlignedSymbolsSpan`. On the other hand, each class can have it's own lib if they are only used in a standalone way, like most of the other utility classes in this package and their corresponding libs in this BUILD file.